spec/liquid_tags/unified_embed/registry_spec.rb

```diff
@@ -106,6 +112,13 @@
         .to eq(GistTag)
     end
 
+    it "returns GlitchTag for a valid glitch url" do
```

Comment:
Suggestion: we generally use this metadata when a single spec has more than one `expect`. This is the case here, even if somewhat hidden in the `each`:

```suggestion
    it "returns GlitchTag for a valid glitch url", :aggregate_failures do
```